src/Files.App/Filesystem/ListedItem.cs

```diff
@@ -49,6 +49,9 @@ public string ItemTooltipText
 				return $"{"ToolTipDescriptionName".GetLocalizedResource()} {Name}{Environment.NewLine}" +
 					$"{"ToolTipDescriptionType".GetLocalizedResource()} {itemType}{Environment.NewLine}" +
 					$"{"ToolTipDescriptionDate".GetLocalizedResource()} {ItemDateModified}" +
+					(!String.IsNullOrWhiteSpace(FileSize)
+						? $"{Environment.NewLine}{"ToolTipDescriptionSize".GetLocalizedResource()} {FileSize}"
+						: string.Empty) +
```

Comment:
Worth using a string builder here.

Reply:
It's using the exiting property, so I think it's okay. I'd rather leave a refactor to a separate PR so as not to overcomplicate it right now.

Reply:
> Worth using a string builder here.

I agree. And also, can you replace `String.IsNullOrWhiteSpace` with `string.IsNullOrWhiteSpace`?

Reply:
See new comment.